.github/workflows/image-arm.yaml

```diff
@@ -47,7 +49,13 @@ jobs:
           MODEL: ${{ matrix.model }}
         run: |
           export TAG=${GITHUB_REF##*/}
-          ./earthly.sh +all-arm --IMAGE_NAME=kairos-$FLAVOR-$TAG.img --IMAGE=quay.io/kairos/kairos-$FLAVOR:$TAG --MODEL=$MODEL --FLAVOR=$FLAVOR
+          export IMAGE=quay.io/kairos/kairos-$FLAVOR
+
+          ./earthly.sh +all-arm --IMAGE_NAME=kairos-$FLAVOR-$TAG.img --IMAGE="$IMAGE:$TAG" --MODEL=$MODEL --FLAVOR=$FLAVOR
+
+          docker push "$IMAGE:$TAG" # Otherwise .RepoDigests will be empty for some reason
```

Comment:
the problem here is that we always push. we should just push on master

Reply:
If we are not pushing, why are we signing?

Reply:
added condition to sign and push only on `main`